releases/1.1.1-SNAPSHOT.yml

```diff
@@ -0,0 +1,3 @@
+version: 1.1.1-SNAPSHOT
+rpm_version: 1.1.1~SNAPSHOT
+deb_version: 1.1.1~SNAPSHOT
```

Comment:
Oh, didn't know that some RHEL version don't like that. Hopefully RHEL before 6?